CHANGELOG.md

```diff
@@ -1,5 +1,101 @@
+* add deserializer for SqlType ([#4830](https://github.com/confluentinc/ksql/pull/4830)) ([eed9912](https://github.com/confluentinc/ksql/commit/eed99123a7f673eec9c1f5224c164a5737e6dc41))
+* allow trailing slash in listeners config (MINOR) ([#5012](https://github.com/confluentinc/ksql/pull/5012)) ([13b0455](https://github.com/confluentinc/ksql/commit/13b04557d01aed36dcf11e8d6c8300dae4143615))
+* Allows unclosed quote characters to exist in comments ([#4993](https://github.com/confluentinc/ksql/pull/4993)) ([fd65021](https://github.com/confluentinc/ksql/commit/fd6502115170c97f6a2c61cabf513145ea9c4cdc))
+* avoid duplicate column name errors from auto-generated aliases ([#4827](https://github.com/confluentinc/ksql/pull/4827)) ([258d0b0](https://github.com/confluentinc/ksql/commit/258d0b06d81954af2c4e2c17fd7f26c74f315368))
+* avoid long possible format lists on PRINT TOPIC for nulls ([#4867](https://github.com/confluentinc/ksql/pull/4867)) ([a66e489](https://github.com/confluentinc/ksql/commit/a66e489d176e1ed6ea715af3d9cfd75a351e0447))
+* better error code on shutdown ([#4754](https://github.com/confluentinc/ksql/pull/4754)) ([17d758d](https://github.com/confluentinc/ksql/commit/17d758d0137102aa30493eee458e1a7c2ea54f82))
+* Catch server startup exceptions ([#4974](https://github.com/confluentinc/ksql/pull/4974)) ([898f3a1](https://github.com/confluentinc/ksql/commit/898f3a104f88fba5cf79e02f12ef43760065eaae))
+* CommandRunner metric has correct metric displayed when thread dies ([#4653](https://github.com/confluentinc/ksql/pull/4653)) ([1db542b](https://github.com/confluentinc/ksql/commit/1db542bfea5006fc9b3af59bca28d4af48fc5d57))
+* do not allow GROUP BY and PARTITION BY on boolean expressions ([#4940](https://github.com/confluentinc/ksql/pull/4940)) ([d84d2d1](https://github.com/confluentinc/ksql/commit/d84d2d110d5ac502de4ba16602aa32babe24b4ca))
+* do not allow grouping sets ([#4942](https://github.com/confluentinc/ksql/pull/4942)) ([51bb9f6](https://github.com/confluentinc/ksql/commit/51bb9f6f4bcef2d75f6d6d96c4c883559a21999b))
+* do not allow implicit casting in UDAF function lookup ([#5145](https://github.com/confluentinc/ksql/pull/5145)) ([6709010](https://github.com/confluentinc/ksql/commit/6709010b06fed487334338e5e102bf53d51b4c46))
+* do not filter out rows where PARTITION BY resolves to null ([#4823](https://github.com/confluentinc/ksql/pull/4823)) ([e75a792](https://github.com/confluentinc/ksql/commit/e75a79286927a6e43b7d580b32215fb18b00d707))
+* Don't set chunked transfer encoding for HTTP2 proxy ([#4985](https://github.com/confluentinc/ksql/pull/4985)) ([3f16844](https://github.com/confluentinc/ksql/commit/3f1684400dfcdc8d1daafe8feb9363efed9327d8))
+* Don't wait for streams thread to be in running state ([#4908](https://github.com/confluentinc/ksql/pull/4908)) ([ceeead2](https://github.com/confluentinc/ksql/commit/ceeead21d32232a2e30d79b84342a0dec3c36fd4))
```

Comment:
This can be dropped. It's present in 0.8.1.

Reply:
done